### PR TITLE
Tweak cosmetics of the DAOs page

### PR DESCRIPTION
**Pull request by tibetsprague**
Line up create button with right of the daos list.
Make search box font size smaller
Make the search box sticky
Improve how we handle stickiness
Get rid of empty space
Definitely check mobile!

Fixes https://github.com/daostack/alchemy/issues/1623

### CODE REVIEW

File: src/components/Daos/Daos.scss [hunk 1]
```diff
@@ -19,29 +20,31 @@
...
     padding: 2px;
   }
 }
 
 .createDaoButton {
-  position: fixed;
-  right: 20px;
-  top: 122px;
+  position: sticky;
+  float: right;
```
Comment (dkent600) [inline src/components/Daos/Daos.scss:46]: We can pretty much always do things better without using float.  Dunno what you're attempting here, but can it not be done with css flex styles?

File: src/components/Daos/DaosPage.tsx [hunk 2]
```diff
@@ -48,17 +49,31 @@ class DaosPage extends React.Component<IProps, IState> {
     super(props);
 
     this.state = {
+      isMobile: window.innerWidth <= 550,
```
Comment (dkent600) [inline src/components/Daos/DaosPage.tsx:52]: wow this is starting to look like a big hack, definitely not a "tweak".  Can this not be done with css and @media rules?

File: src/components/Daos/Daos.scss [hunk 3]
```diff
@@ -19,29 +20,31 @@
...
     width: 425px;
     padding: 2px;
   }
 }
 
 .createDaoButton {
-  position: fixed;
-  right: 20px;
-  top: 122px;
+  position: sticky;
```
Comment (dkent600) [inline src/components/Daos/Daos.scss:45]: Sticky has sketchy browser support:  https://caniuse.com/#feat=css-sticky

File: src/components/Daos/Daos.scss (hunk 1 above)
Comment (tibetsprague) [reply src/components/Daos/Daos.scss:46]: maybe it could. but this works

File: src/components/Daos/DaosPage.tsx (hunk 2 above)
Comment (tibetsprague) [reply src/components/Daos/DaosPage.tsx:52]: not this part no because its in the react code

File: src/components/Daos/Daos.scss (hunk 3 above)
Comment (tibetsprague) [reply src/components/Daos/Daos.scss:45]: hmm. well, is it good enough @EzraWeller ?

**Pull request merged**